### DISCUSSION TITLE
AI buys extreme amount of transport planes

### DISCUSSION THREAD

**Discussion by jrcnn** [Khopa/dcs_liberation#1876] [Q&A] [answered]
In each turn, the enemy AI spends all their money buying IL-76 transport planes and doesn't use them at all
![Untitled](https://user-images.githubusercontent.com/96748321/147505267-f2241d6f-f932-4cd2-8bf9-6c87f0c7755e.png)
.
So I wonder, do transport planes like the C-17 have any use at all in Liberation? How is it used?
Edit: this happened in my custom Persian Gulf campaign, I do not know whether it's the case in the default ones or not. If this issue is unique to my campaign though, what is the fix for that
[iran_campaign.txt](https://github.com/dcs-liberation/dcs_liberation/files/7781756/iran_campaign.txt)
?

**Comment by Starfire13** [comment]
Never seen them do that in the standard campaigns. Not sure why they are doing it in your custom one, but perhaps your ground or sea transport routes are not set up properly. You can get around it by just removing the transport squadron completely. At the moment, air transport squadrons basically don't do anything as long as ground or sea transport routes are pretty much always available.

**Comment by RndName** [comment, marked as answer]
Is there a transport route between Bandar Abbas and the other Control Points? AI will only buy the Transport planes when there is no other way to transfer Ground Units (Tanks etc.) to the front line. Maybe there is a problem in the supply routes you defined. Also check if there are factories to produce ground units :)
Have seen such a behaviour some time ago when there was no ground supply route.

**Reply by RndName** [reply]
https://github.com/dcs-liberation/dcs_liberation/wiki/Custom-Campaigns#supply-routes

**Reply by jrcnn** [reply]
It was the shipping lanes. I failed to identify a port area in the Persian Gulf.

**Reply by RndName** [reply]
Glad to hear that you found the root cause! Great :)
